templates/web/base/report/new/form_public_councils_text.html

```diff
@@ -0,0 +1,11 @@
+[% IF c.cobrand.moniker != 'tfl' %]
```

Comment:
I thought it might be easier to explain by showing and not telling, so have pushed a suggestion (e6f78002a8521a34468e7c7667c4fd79a7df8f9a) which uses blocks to remove the `tfl` references from the base template.

Reply:
Ta; I've removed one unneeded tfl bit of that and brought it in.